core-feature-pack/common/src/main/resources/content/bin/systemd/generate_systemd_unit.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# This script is just a helper to generate the Systemd Unit Files for WildFly
+# assuming that the server home will be the one that hold this script file.
+# It also allows to specify the user and group that will run the server.
+
+SCRIPT_NAME=$(basename "$0")
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+JBOSS_HOME="$(realpath "$SCRIPT_DIR/../..")"
+JBOSS_SYSTEMD_MODE="${1}"
+
+if [ -z "$JBOSS_SYSTEMD_MODE" ]; then
+  echo "INFO: Usage: "
+  echo "${SCRIPT_NAME} standalone|domain [user name] [group name]"
+  exit 0;
+fi
+
+SYSTEMD_FILE="$SCRIPT_DIR/wildfly-${JBOSS_SYSTEMD_MODE}.service"
+JBOSS_SYSTEMD_USER="${2:-wildfly}"
+JBOSS_SYSTEMD_GROUP="${3:-wildfly}"
+
+echo "INFO: Systemd Unit File to generate: ${SYSTEMD_FILE}"
+echo "INFO: Using JBOSS_HOME: $JBOSS_HOME"
```

Comment:
As we have JBOSS_HOME here, we should use it at the beginning, to not repeat /opt/wildfly all the time.

Also: 
`echo "INFO: Using JBOSS_HOME: ${JBOSS_HOME}"`

Reply:
> As we have JBOSS_HOME here, we should use it at the beginning, to not repeat /opt/wildfly all the time.

JBOSS_HOME is initialized at the beginning of the script and there is no "/opt/wildfly" path hardcoded in this script, so I don't follow you here.

> Also:
> echo "INFO: Using JBOSS_HOME: ${JBOSS_HOME}"

Done